<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCefViewTestClass</class>
 <widget class="QMainWindow" name="QCefViewTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCefViewTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widgetNavigationbar" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonGoBack">
         <property name="text">
          <string>Go Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonGoForward">
         <property name="text">
          <string>Go Forward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonReloadOrStop">
         <property name="text">
          <string>Reload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUrl"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonTriggerEvent">
         <property name="text">
          <string>Trigger Event</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonTransparent">
         <property name="text">
          <string>Transparent</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>231</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCefView" name="widgetCef" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCefView</class>
   <extends>QWidget</extends>
   <header>qcefview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qcefviewtest.qrc"/>
 </resources>
 <connections/>
</ui>
